spec/jobs/deposit_job_spec.rb

```diff
@@ -13,11 +13,27 @@
       content_type: 'image/svg+xml'
     )
   end
+  let!(:blob2) do
+    ActiveStorage::Blob.create_and_upload!(
+      io: File.open(Rails.root.join('spec/fixtures/files/sul.svg')),
+      filename: 'sul2.svg',
+      content_type: 'image/svg+xml'
+    )
+  end
   let(:attached_file) { build(:attached_file) }
+  let(:attached_file2) { build(:attached_file) }
   let(:work) { build(:work, collection: collection, assign_doi: false) }
-  let(:work_version) do
-    build(:work_version, id: 8, work: work, attached_files: [attached_file], version_description: 'Updated metadata')
+  let(:first_work_version) do
+    build(:work_version, work: work, attached_files: [attached_file], version: 1)
   end
+  let(:second_work_version_metadata_only) do
```

Comment:
Can this be moved down into the context "when files have not changed"?

Reply:
Done.